torch/onnx/_internal/exporter.py

```diff
@@ -96,6 +115,16 @@ def resolve(value: Optional[T], fallback: Union[T, Callable[[], T]]) -> T:
 
         self.opset_version = resolve(options.opset_version, _DEFAULT_OPSET_VERSION)
         self.dynamic_shapes = resolve(options.dynamic_shapes, False)
+        # TODO: Prevent circular dep + [onnx]script dep
+        import torch.onnx._internal.fx.dynamo_graph_extractor as dynamo_graph_extractor
+        from torch.onnx._internal.fx import (  # TODO: onnxscript is not available on Pytorch CI yet
```

Comment:
well it is available on PyTorch onnx CI, but will take quite a while, if say possible, to be available on PyTorch non-onnx CI. I guess this comment is meant for the latter. But not sure why CI needs to be mentioned explicitly. It doesn't make sense for PyTorch non-onnx CI to have onnxscript if onnxscript is not a dependency of PyTorch.

Reply:
There is a limitation on maximum line length, so we can't be too verbose.
It is assumed who reads this comment knows they are dealing with `torch.onnx`-only namespace already, right?

Reply:
My point is the real problem is already mentioned as `[onnx]script dep` up there. I don't understand what is special about "CI" that it needs to be brought up.

Reply:
sorry, dude, the rephrasing somehow was `git add`ed on my last commit.

The comment will now read:
```python
import torch.onnx._internal.fx.dynamo_graph_extractor as dynamo_graph_extractor  # TODO: Prevent circular dep
from torch.onnx._internal.fx import (  # TODO: PyTorch does not take dep on onnxscript outside torch.onnx context
```